pkg/rancher-desktop/backend/images/imageProcessor.ts

```diff
@@ -255,20 +249,23 @@ export abstract class ImageProcessor extends EventEmitter {
    */
   async processChildOutput(child: ChildProcess, subcommandName: string, sendNotifications: boolean, args?: string[]): Promise<childResultType> {
     const result = { stdout: '', stderr: '' };
+    const commandName = args ? 'trivy' : this.processorName;
```

Comment:
This is going to be so confusing when we start trying to use args for one of the non-trivy commands…

Reply:
Yeah, but you won't. This is already a hack. args for other commands are not passed in at all, except for `subcommandName`, which is needed for another hack to treat `nerdctl images` differently.

Reply:
No, `subcommandName` (per the method docs) is for displaying to the user; the fact that it's used for `nerdctl images` is the hack, but we still need to preserve it for its primary purpose.

A better way to do this would be to just pass it in:
```ts
type ProcessChildOutputOptions = {
  commandName?: string;
  subcommandName: string;
  sendNotifications: boolean;
}
async processChildOutput(child: ChildProcess, options: ProcessChildOutputOptions, args?: string[]): Promise<childResultType> {
    const commandName = options.commandName ?? this.processorName;
```